about.html

```diff
@@ -0,0 +1,98 @@
+        </div>
+        <div class="about-contact">
+            <h3 class="first-line">Please contact us per Email for nay further questions about Kano State Tech Summit 2015!</h3>
+            <h3 class="second-line">techsummit2015@kano.org</h3>
+        </div>
+    </div>
+    <ul class="creative-common-section">
+        <li class="creative-common-text-one">Kano State <br> Tech Summit 2015 Logo</li>
+        <li class="creative-common-text-one-desktop">Kano State Tech Summit 2015 Logo</li>
+        <i class="fa-solid fa-horizontal-rule line-red-about"></i>
+        <li class="creative-common-text-two">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.</li>
+        <li class="creative-common-text-two-desktop">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+            After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+            After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:</li>
+        <li><img src="./img/global-summit-logo.png" alt="" class="global-summit-logo"></li>
+    </ul>
+    <div class="see-the-past">
+        <ul class="see-the-past-items">
+            <li class="see-the-past-text-one">See the past Kano State Tech Summits</li>
+            <i class="fa-solid fa-horizontal-rule line-red-about"></i>
+            <li class="see-the-past-text-two">Take a look at the past two Kano State Tech Summits which took place in Dawanau and in Hadejia</li>
+        </ul>
+    </div>
+    <div class="past-images">
+        <img src="./img/past-image-1.svg" alt="" class="see-the-past-pic">
+        <img src="./img/past-image-2.svg" alt="" class="see-the-past-pic">
+    </div>
+    <div class="partner">
+        <div class="partner-header">
+            <h3 class="partner-text">Partners</h3>
+            <i class="fa-solid fa-horizontal-rule fa-horizontal-rule-red"></i>
+        </div>
+        <div class="partnet-companies">
+            <ul class="partner-company-names">
+                <li><img src="./img/mozilla 1.svg" alt="" class="company-images"></li>
+                <li><img src="./img/googlekorea 1.svg" alt="" class="company-images"></li>
+                <li><img src="./img/naver 1.svg" alt="" class="company-images"></li>
+                <li><img src="./img/daumkakaologo 1.svg" alt="" class="company-images"></li>
+                <li><img src="./img/airbnb 1.svg" alt="" class="company-images"></li>
+            </ul>
+        </div>
+    </div>
+    <footer>
+        <div class="footer">
+            <img class="global-minbar-logo" src="./img/global_minbar.png" alt="global minbar logo">
+            <h3 class="footer-text">2023 Global Minbar Creative Design. Some Rights Reserved.</h3>
+        </div>
+    </footer>
```

Comment:
- [ ] From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see, on the about page, your footer background color does not match [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). 
- [ ] Also, the partner's section is not present on the desktop so make sure that you remove it on the desktop and maintain it on mobile to match the design.

| Your Design |Cindy Design |
| :---:   | :---: | 
| ![Screenshot from 2023-03-18 20-45-41](https://user-images.githubusercontent.com/92303891/226138666-ac0155d1-be98-4ae0-961b-699f10aabad1.png) | ![image](https://user-images.githubusercontent.com/92303891/225560636-3146fbf9-1834-4127-99fe-d07f021ed663.png)  |